<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>457</width>
    <height>305</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QToolButton" name="Start_toolButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>190</y>
      <width>51</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string>摄像人脸检测</string>
    </property>
    <property name="icon">
     <iconset resource="qrc.qrc">
      <normaloff>:/new/prefix1/images/face_recognition_scan_64px_1169929_easyicon.net.png</normaloff>:/new/prefix1/images/face_recognition_scan_64px_1169929_easyicon.net.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>52</width>
      <height>52</height>
     </size>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="Photo_toolButton">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>190</y>
      <width>51</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string>图片检测</string>
    </property>
    <property name="icon">
     <iconset resource="qrc.qrc">
      <normaloff>:/new/prefix1/images/landscape_1195577_easyicon.net.svg</normaloff>:/new/prefix1/images/landscape_1195577_easyicon.net.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>52</width>
      <height>52</height>
     </size>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="Save_toolButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>190</y>
      <width>51</width>
      <height>45</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="text">
     <string>图片人脸头像保存</string>
    </property>
    <property name="icon">
     <iconset resource="qrc.qrc">
      <normaloff>:/new/prefix1/images/file_save_474px_1173837_easyicon.net.png</normaloff>:/new/prefix1/images/file_save_474px_1173837_easyicon.net.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>52</width>
      <height>52</height>
     </size>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="Collect_toolButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>190</y>
      <width>61</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string>摄像头人脸采集</string>
    </property>
    <property name="icon">
     <iconset resource="qrc.qrc">
      <normaloff>:/new/prefix1/images/metering_evaluative_1195580_easyicon.net.svg</normaloff>:/new/prefix1/images/metering_evaluative_1195580_easyicon.net.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>52</width>
      <height>52</height>
     </size>
    </property>
    <property name="autoRaise">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="Train_toolButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>190</y>
      <width>61</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string>训练</string>
    </property>
    <property name="icon">
     <iconset resource="qrc.qrc">
      <normaloff>:/new/prefix1/images/training_plan_256px_1146065_easyicon.net.png</normaloff>:/new/prefix1/images/training_plan_256px_1146065_easyicon.net.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>52</width>
      <height>52</height>
     </size>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="Recface_toolButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>190</y>
      <width>51</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string>摄像识别</string>
    </property>
    <property name="icon">
     <iconset resource="qrc.qrc">
      <normaloff>:/new/prefix1/images/face_detect.png</normaloff>:/new/prefix1/images/face_detect.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>54</width>
      <height>54</height>
     </size>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>457</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="qrc.qrc"/>
 </resources>
 <connections/>
</ui>
